django_ses/utils.py

```diff
@@ -224,7 +224,11 @@ def _get_bytes_to_sign(self):
 
         bytes_to_sign = []
         for field in fields_to_sign:
-            bytes_to_sign.append(f"{field}\n{self._data[field]}\n")
+            field_value = self._data.get(field)
+            if field_value:
```

Comment:
I'm a fan of patterns like:

```python
if not X:
    continue  # or return
bytes_to_sign.append(...
```

Mostly because it reduces nesting on branchy code. But it's a preference.

Reply:
Me too. I didn't do so here because it's not branchy or a long if statement, but I'll do another PR with it.